app/routers/api/browser.py

```
@@ -17,14 +17,15 @@
 
 router = APIRouter(prefix="/api/v1/browser", tags=["browser"])
 logger = logging.getLogger(__name__)
-
-
+from typing import Annotated, Dict
+...
```

Ruff (0.15.9)

[warning] Unnecessary `...` literal

Remove unnecessary `...`

(PIE790)